bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml

```diff
@@ -5,123 +5,221 @@
+			</options>
+			<default>MIN</default>
+		</parameter>
+		<parameter name="period" type="text">
+			<label>Period</label>
+			<description>Observation period for the queried value.</description>
+			<options>
+				<option value="30min">30 minutes</option>
+				<option value="1hour">1 hour</option>
+				<option value="3hours">3 hours</option>
+				<option value="1day">This day</option>
+				<option value="1week">This week</option>
+				<option value="1month">This month</option>
+			</options>
+			<default>1week</default>
 		</parameter>
 	</config-description>
 
-	<config-description uri="thing-type:netatmo:module">
-		<parameter name="id" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})" required="true">
-			<label>Module ID</label>
-			<description>Id of the Module</description>
+	<config-description uri="netatmo:co2:config">
+		<parameter name="limit" type="text">
+			<label>Boundary</label>
+			<description>Defines the requested boundary.</description>
+			<options>
+				<option value="MIN">Minimum</option>
+				<option value="MAX">Maximum</option>
+			</options>
+			<default>MIN</default>
+		</parameter>
+		<parameter name="period" type="text">
+			<label>Period</label>
+			<description>Observation period for the queried value.</description>
+			<options>
+				<option value="30min">30 minutes</option>
+				<option value="1hour">1 hour</option>
+				<option value="3hours">3 hours</option>
+				<option value="1day">This day</option>
+				<option value="1week">This week</option>
+				<option value="1month">This month</option>
+			</options>
+			<default>1week</default>
 		</parameter>
+	</config-description>
 
-		<parameter name="parentId" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})" required="true">
-			<label>Device ID</label>
-			<description>Id of the main device</description>
+	<config-description uri="netatmo:noise:config">
+		<parameter name="limit" type="text">
+			<label>Boundary</label>
+			<description>Defines the requested boundary.</description>
+			<options>
+				<option value="MIN">Minimum</option>
+				<option value="MAX">Maximum</option>
+			</options>
+			<default>MIN</default>
+		</parameter>
+		<parameter name="period" type="text">
+			<label>Period</label>
+			<description>Observation period for the queried value.</description>
+			<options>
+				<option value="30min">30 minutes</option>
+				<option value="1hour">1 hour</option>
+				<option value="3hours">3 hours</option>
+				<option value="1day">This day</option>
+				<option value="1week">This week</option>
+				<option value="1month">This month</option>
+			</options>
+			<default>1week</default>
 		</parameter>
 	</config-description>
 
-	<config-description uri="thing-type:netatmo:natherm1">
-		<parameter name="id" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})" required="true">
-			<label>Module ID</label>
-			<description>Id of the Module</description>
+	<config-description uri="netatmo:hum:config">
```

Comment:
You defined 5 times the same configuration: netatmo:temp:config, netatmo:pressure:config, netatmo:co2:config, netatmo:noise:config and netatmo:hum:config.
Why not one unique configuration ?

Reply:
Ok, changed.